src/definition.rs

```diff
@@ -0,0 +1,49 @@
+use std::path::Path;
+
+use tower_lsp::lsp_types::*;
+
+use crate::{file_utils::resolve_bazelrc_path, line_index::IndexEntryKind, parser::Line};
+
+pub fn get_definitions(
+    file_path: &Path,
+    kind: &IndexEntryKind,
+    line: &Line,
+) -> Option<GotoDefinitionResponse> {
+    match kind {
+        IndexEntryKind::FlagValue(flag_nr) => {
+            let flag = &line.flags[*flag_nr];
+            if Option::is_none(&line.command) {
+                return None;
+            }
+            if line.flags.len() != 1 {
+                return None;
+            }
+            let command_name = &line.command.as_ref().unwrap().0;
+            if *command_name != "import" && *command_name != "try-import" {
+                return None;
+            }
```

Comment:
By using the Elvis operator, we should be able to avoid the `Option::is_none(&line.command)` check

```suggestion
            let flag = &line.flags[*flag_nr];
            let command_name = &line.command?.0;
            if *command_name != "import" && *command_name != "try-import" {
                return None;
            }
            
            if line.flags.len() != 1 {
                return None;
            }
```